private-templates-service/client/src/components/NavBar/SearchBar/SearchBar.tsx

```diff
@@ -0,0 +1,92 @@
+import React, {useState, useEffect} from 'react';
+import {StyledSearchBox} from './styled';
+import { RootState } from '../../../store/rootReducer';
+import { connect } from 'react-redux';
+import { search, clearSearch } from '../../../store/search/actions';
+import { initializeIcons } from 'office-ui-fabric-react/lib/Icons';
+import {THEME, BREAK} from '../../../globalStyles';
```

Comment:
can we have a leading and trailing space around the two elements being imported `{ THEME, BREAK }`